### PR TITLE
Allow sending email from the new domain when switched on

### PR DESCRIPTION
**Pull request by mattandrews**
Once we switch over to the new domain, we can start sending emails from it too.

### CODE REVIEW

File: services/mail.js [hunk 1]
```diff
@@ -48,10 +48,10 @@ function generateHtmlEmail({ template, templateData }) {
                 juice.juiceResources(
                     html,
                     { removeStyleTags: false, webResources: { relativeTo: publicRoot } },
-                    function(juceErr, newHtml) {
-                        /* istanbul ignore if  */
-                        if (juceErr) {
-                            reject(juceErr);
+                    function(juiceErr, newHtml) {
```
Comment (mattandrews) [inline services/mail.js]: (the typo was annoying me)

**Comment by mattandrews** [comment]
Have tested this locally and can confirm it works:

image

image

File: services/mail.js (hunk 1 above)
Comment (davidrapson) [reply services/mail.js]: I leave thme jsut fro you

Comment (davidrapson) [reply services/mail.js]: (thanks for fixing this though)

**Pull request merged**